resources/apigateway-domainnames.go

```diff
@@ -76,9 +92,20 @@ func (f *APIGatewayDomainName) Remove(ctx context.Context) error {
 }
 
 func (f *APIGatewayDomainName) Properties() types.Properties {
-	return types.NewPropertiesFromStruct(f)
```

Comment:
Please revert back to `types.NewPropertiesFromStruct(f)` this will automatically detect the `Tags map[string]string` and add them appropriately to the resource.